tests/integration-tests/tests/performance_tests/test_starccm.py

```diff
@@ -0,0 +1,95 @@
+import json
+import logging
+
+import boto3
+import pytest
+from assertpy import assert_that
+
+from remote_command_executor import RemoteCommandExecutionError, RemoteCommandExecutor
+
+# timeout in seconds
+STARCCM_INSTALLATION_TIMEOUT = 1800
+STARCCM_JOB_TIMEOUT = 600
+STARCCM_LICENCE_SECRET = "perf_integ_secret"
+TASK_VCPUS = 36  # vCPUs are cut in a half because multithreading is disabled
+BASELINE_ELAPSED_TIME = {8: 64.475, 16: 33.1723, 32: 17.8983}
```

Comment:
We know that this dictionary has cluster size as key and elapsed time in seconds as value.
Nevertheless, this should be clear: or with a comment saying that, oir with the name of the dictionary (e.g. `BASELINE_CLUSTER_SIZE_ELAPSED_SECONDS`)